pkg/assets/isto-workspace-deploy.go

```diff
@@ -0,0 +1,365 @@
+// Package assets Code generated by go-bindata. (@generated) DO NOT EDIT.
```

Comment:
What's the difference between this and "pkg/assets/generated/isto-workspace-deploy.go"? They seem pretty dup

Reply:
Seems to me like leftover before tweaking go-bindata Make target. Fixed 72ac2c3